purchases/src/androidTestIntegrationTest/kotlin/com/revenuecat/purchases/helpers/BillingAbstractMockHelper.kt

```diff
@@ -3,12 +3,13 @@ package com.revenuecat.purchases.helpers
 import com.revenuecat.purchases.ProductType
 import com.revenuecat.purchases.common.BillingAbstract
 import com.revenuecat.purchases.common.StoreProductsCallback
+import com.revenuecat.purchases.factories.StoreProductFactory
 import com.revenuecat.purchases.models.StoreProduct
 import io.mockk.every
 import io.mockk.slot
 
 fun BillingAbstract.mockQueryProductDetails(
-    queryProductDetailsSubsReturn: List<StoreProduct> = emptyList(),
+    queryProductDetailsSubsReturn: List<StoreProduct> = listOf(StoreProductFactory.createGoogleStoreProduct()),
```

Comment:
Since we were defaulting to this product in many places, I just moved it here. I checked and all other places are overriding this property so nothing else should change.